### DISCUSSION TITLE
How do I add two keymaps to correspond to the two opener？

### DISCUSSION THREAD

**Discussion by blankPen** [sxyazi/yazi#489] [1. Q&A] [answered]
I want to open the file in different ways with two different keys, as shown below
```
keymaps = [
	{ on = [ "o", "v" ],         exec = "??",    desc = "Open with vim" },
	{ on = [ "o", "c" ],         exec = "??,     desc = "Open with vscode" },
]
```

**Comment by sxyazi** [comment, marked as answer]
I think you can call vim or vscode directly, without following the opener rules. Like this:

```bash
keymaps = [
	{ on = [ "o", "v" ], exec = """  shell --confirm 'vim "$@"'  """ },
	{ on = [ "o", "c" ], exec = """  shell --confirm 'code "$@"'  """ }
]
```

The `shell` command docs here: https://yazi-rs.github.io/docs/configuration/keymap#:~:text=without%20the%20extension.-,shell,-%3A%20Run%20a%20shell
